### PR TITLE
Fixed Card List Positioning

### PR DESCRIPTION
**Pull request by JA-26-01**
## Changes proposed

This pull request addressed issue 1 in Enhancements, which was to shift the card list to the center for better visuals.

## Screenshots

Older layout:

image

Proposed layout:

1. with many elements

image

2. with a single row of elements

image

3. with a single card

image

4. with a single element on new row

image

5. for smaller screens

image

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hey it's breaking down:

image

**Comment by JA-26-01** [comment]
can you tell me what exactly are you facing? Like breaking down as in?

**Comment by rupali-codes** [comment]
> can you tell me what exactly are you facing? Like breaking down as in?

It's showing a scrollbar for small screen (14 inches) also the color is breaking down

**Comment by rupali-codes** [comment]
> > can you tell me what exactly are you facing? Like breaking down as in?
> 
> Looks like we have overflow along X as it seems from the screenshot

Yeah, it needs to be fixed for different screen, as one may not work for each screen

**Comment by JA-26-01** [comment]
> > can you tell me what exactly are you facing? Like breaking down as in?
> 
> Looks like we have overflow along X as it seems from the screenshot



> > > can you tell me what exactly are you facing? Like breaking down as in?
> > 
> > 
> > Looks like we have overflow along X as it seems from the screenshot
> 
> Yeah, it needs to be fixed for different screens, as one may not work for each screen

Oh okay got it. Let me look into it.

**Comment by JA-26-01** [comment]
@rupali-codes can you check the new commit once and let me know? I have made the changes and its working fine for me now.

image

**Comment by rupali-codes** [comment]
> > > can you tell me what exactly are you facing? Like breaking down as in?
> > 
> > 
> > Looks like we have overflow along X as it seems from the screenshot
> 
> > > > can you tell me what exactly are you facing? Like breaking down as in?
> > > 
> > > 
> > > Looks like we have overflow along X as it seems from the screenshot
> > 
> > 
> > Yeah, it needs to be fixed for different screens, as one may not work for each screen
> 
> Oh okay got it. Let me look into it.

Nope, 
image

I guess it would be better if you can re-position the cards in your way, like make it from scratch, just the design should be similar to the current design. What do you think?

**Comment by JA-26-01** [comment]
> 

Okay sure. Let me try it.

File: components/Cards/CardsList.tsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ const CardsList:FC<{ cards: IData[] }> = (props) => {
           />
         ))}
       </ul>
+      </div>
```
Comment (CBID2) [inline components/Cards/CardsList.tsx:37]:
```suggestion
      </article>
```

Comment (CBID2) [inline components/Cards/CardsList.tsx:37]:
```suggestion
      </article>
```

File: components/Cards/CardsList.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ const CardsList:FC<{ cards: IData[] }> = (props) => {
 
   return (
     <>
+    <div className="cardlist">
```
Comment (CBID2) [inline components/Cards/CardsList.tsx:27]:
```suggestion
    <article className="cardlist">
```

Comment (CBID2) [inline components/Cards/CardsList.tsx:27]:
```suggestion
    <article className="cardlist">
```

Comment (CBID2) [inline components/Cards/CardsList.tsx:27]:
```suggestion
    <article className="cardlist">
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @JA-26-01! 😄 I've made some suggestions on how to make your cards more semantic. I hope you find them helpful! 😄 Also, avoid pushing the PR to the main branch. It'll make the PR process difficult. Other than that, well done! 😄

**Comment by JA-26-01** [comment]
> Hi @JA-26-01! 😄 I've made some suggestions on how to make your cards more semantic. I hope you find them helpful! 😄 Also, avoid pushing the PR to the main branch. It'll make the PR process difficult. Other than that, well done! 😄

Oh thank you so much. This looks much more structured. And yes, will keep that it mind! Actually my exams are going on so could not give it a go right now.

**Comment by rupali-codes** [comment]
> > Hi @JA-26-01! 😄 I've made some suggestions on how to make your cards more semantic. I hope you find them helpful! 😄 Also, avoid pushing the PR to the main branch. It'll make the PR process difficult. Other than that, well done! 😄
> 
> Oh thank you so much. This looks much more structured. And yes, will keep that it mind! Actually my exams are going on so could not give it a go right now.

Sure, you can take your time and good luck for the exams :)

**Comment by CBID2** [comment]
Hey @JA-26-0! 😄 Are you still working on this?

**Comment by rupali-codes** [comment]
Closing this PR because of inactivity, feel free to open a new PR whenever you are back!

**Pull request closed without merging**